src/lando/utils/github.py

```diff
@@ -113,6 +117,25 @@ def get_patch(self, pull_number: int) -> str:
             headers={"Accept": "application/vnd.github.patch"},
         )
 
+    def open_pull_request(self, pull_number: int) -> dict:
+        """Open the given pull request."""
+        return self._post(
+            f"{self.repo_base_url}/pulls/{pull_number}", json={"state": "open"}
+        )
+
+    def close_pull_request(self, pull_number: int) -> dict:
+        """Close the given pull request."""
+        return self._post(
+            f"{self.repo_base_url}/pulls/{pull_number}", json={"state": "closed"}
+        )
+
+    def add_comment_to_pull_request(self, pull_number: int, comment: str) -> dict:
+        """Add a comment to the given pull request."""
+        return self._post(
+            f"{self.repo_base_url}/issues/{pull_number}/comments",
+            json={"body": comment},
+        )
```

Comment:
I'd move this to the PullRequest object, but this probably needs further discussion on which way we want to do this.